_tuning-your-cluster/availability-and-recovery/remote-store/migrating-to-remote.md

```diff
@@ -0,0 +1,205 @@
+   ```json
+   GET "/_cluster/health?pretty"
+   ```
+   The response should look similar to the following example:
+   ```json
+   {
+       "cluster_name":"opensearch-dev-cluster",
+       "status":"green",
+       "timed_out":false,
+       "number_of_nodes":4,
+       "number_of_data_nodes":4,
+       "active_primary_shards":1,
+       "active_shards":4,
+       "relocating_shards":0,
+       "initializing_shards":0,
+       "unassigned_shards":0,
+       "delayed_unassigned_shards":0,
+       "number_of_pending_tasks":0,
+       "number_of_in_flight_fetch":0,
+       "task_max_waiting_in_queue_millis":0,
+       "active_shards_percent_as_number":100.0
+   }
+   ```
+1. Disable shard replication to prevent shard replicas from being created while nodes are being taken offline. This stops the movement of Lucene index segments on nodes in your cluster. You can disable shard replication by querying the `_cluster/settings` API endpoint:
+   ```json
+   PUT "/_cluster/settings?pretty"
+   {
+       "persistent": {
+           "cluster.routing.allocation.enable": "primaries"
+       }
+   }
+   ```
+   The response should look similar to the following example:
+   ```json
+   {
+     "acknowledged" : true,
+     "persistent" : {
+       "cluster" : {
+         "routing" : {
+           "allocation" : {
+             "enable" : "primaries"
+           }
+         }
+       }
+     },
+     "transient" : { }
+   }
+   ```
+
+1. Perform a flush operation on the cluster to commit transaction log entries to the Lucene index:
+   ```json
+   POST "/_flush?pretty"
+   ```
+   The response should look similar to the following example:
+   ```json
+   {
+     "_shards" : {
+       "total" : 4,
+       "successful" : 4,
+       "failed" : 0
+     }
+   }
+   ```
+1. Set the `remote_store.compatibility_mode` to `mixed` to allow remote-store backed nodes to join the cluster
+   ```json
+   PUT "/_cluster/settings?pretty"
+   {
+       "persistent": {
+           "remote_store.compatibility_mode": "mixed"
+       }
+   }
+   ```
+   The response should look similar to the following example:
+   ```json
+   {
+     "acknowledged" : true,
+     "persistent" : { 
+      "remote_store" : {
+      "compatibility_mode" : "mixed"
+      },
+      "transient" : { }
+   }
+   }
+   ```
+2. Review your cluster and identify the first node to upgrade.
+2. Provide the remote store repository details as node attributes in `opensearch.yml`, as shown in the following example.
+
+   ```yml
+   # Repository name
+   node.attr.remote_store.segment.repository: my-repo-1
+   node.attr.remote_store.translog.repository: my-repo-2
+   
+   # Segment repository settings
+   node.attr.remote_store.repository.my-repo-1.type: s3
+   node.attr.remote_store.repository.my-repo-1.settings.bucket: <Bucket Name 1>
+   node.attr.remote_store.repository.my-repo-1.settings.base_path: <Bucket Base Path 1>
+   node.attr.remote_store.repository.my-repo-1.settings.region: us-east-1
```

Comment:
```suggestion
   node.attr.remote_store.repository.my-repo-1.settings.region: <region (example: us-east-1)>
```

Reply:
Following the remote store documentation approach only.